src/Chamilo/CoreBundle/Component/Editor/CkEditor/Toolbar/TestAnswerFeedback.php

```diff
@@ -67,7 +67,7 @@ protected function getMaximizedToolbar()
             ['Cut', 'Copy', 'Paste', 'PasteText', 'PasteFromWord'],
             ['Undo', 'Redo', '-', 'SelectAll', 'Find', '-', 'RemoveFormat'],
             ['Link', 'Unlink', 'Anchor', 'Glossary'],
-            ['Image', 'Mapping', 'Video', 'Oembed', 'Youtube', 'Flash', 'Audio', 'leaflet', 'Smiley', 'SpecialChar'],
+            ['Image', 'Mapping', 'Video', 'Oembed', 'Youtube', 'VimeoEmbed', 'Flash', 'Audio', 'leaflet', 'Smiley', 'SpecialChar'],
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line exceeds maximum limit of 120 characters; contains 131 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5500355)